app/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
@goonerDroid Why is MIT licence needed in the manifest file?

Reply:
@darshanpania I have Added a license script for all files, and this license template will be added during spotlessTask. Do you want me to exclude the license from the manifest(.xml) or remove it completely from .xml files?

Reply:
@goonerDroid See my comment on the other thread below